### PR TITLE
Add and style contact section

### PR DESCRIPTION
**Pull request by izabelastanislawska**

### CODE REVIEW

**Review by szymonlicau — CHANGES_REQUESTED** [review]
I can see you've been busy tidying things up but it's starting to come along quite nicely! 🚀 
I've suggested a few things you could potentially refactor and also I'm not sure I understand one of your choices correctly - could you please elaborate on that one? 🙏

File: src/components/TheScreenings.vue [hunk 1]
```diff
@@ -1,26 +1,36 @@
...
-            :img="movie.poster_url"
-            :genre="movie.genre.name"
-        /> -->
+        
+        <!-- <MovieCard v-model="movieCard">
+            <h3>Title: {{ title }}</h3>
+            <p>Movie length: {{ length }} min.</p>
+            <img :src="img" />
+            <p>Genre: {{ genre }}</p>
+        </MovieCard> -->
```
[the remark this answers is not in the document]
Comment (izabelastanislawska) [reply src/components/TheScreenings.vue:33]: It doesn't work :( I'm doing something wrong, that's why I commented it and I will be working on it later

[the remark this answers is not in the document]
Comment (szymonlicau) [reply src/components/TheScreenings.vue:33]: If you feel like you might need help with this - feel free to hit me up and we'll figure it out 😄

File: src/components/TheScreenings.vue [hunk 2]
```diff
@@ -55,23 +65,45 @@ export default {
...
+    // },
+    formatLength(lengthInMinutes = 0) {
+        const hours = Math.floor(lengthInMinutes/60);
+        const minutes = `0${lengthInMinutes%60}`.slice(-2);
+        return `${hours}:${minutes}`
+    },
+    currentDate() {
+        const currentDate = new Date().toLocaleDateString("en-GB", {weekday:"long", day:"2-digit", month:"2-digit", year:"numeric"});
+        return currentDate;
     }
```
[the remark this answers is not in the document]
Comment (izabelastanislawska) [reply src/components/TheScreenings.vue]: I fixed it, and also committed MovieCard fixes. Can you check?

**Review by szymonlicau — APPROVED** [review]

**Pull request merged**